packages/marketplace/src/components/pages/register.tsx

```diff
@@ -135,7 +135,7 @@ export const Register: React.FunctionComponent<RegisterProps & FormikProps<Regis
             }
             isCenter
           >
-            Check you email to confirm your account
+            <div className="mb-3">Check you email to confirm your account</div>
```

Comment:
Should be "your email" not "you email", sorry!

Reply:
typo updated 👍